### PR TITLE
[4.0] Atum top

### PR DESCRIPTION
**Pull request by ceford**
Pull Request for Issue #27791 

### Summary of Changes

Added customtop jdoc position to atum template.

### Testing Instructions

Obtain test top menu module: https://github.com/ceford/j4xdemos-admin-mod-topmenu/blob/master/mod_topmenu.zip

Install / enable / assign to customtop position

### Expected result

a top menu appears as well as the standard side menu

disable when finished

### Actual result

Works for me

### Documentation Changes Required

None

### CODE REVIEW

**Comment by brianteeman** [comment]
Please look at https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/language/en-GB/tpl_atum.sys.ini and update this PR accordingly

File: administrator/language/en-GB/tpl_atum.sys.ini [hunk 1]
```diff
@@ -15,4 +15,5 @@ TPL_ATUM_POSITION_STATUS="Status"
 TPL_ATUM_POSITION_TITLE="Title"
 TPL_ATUM_POSITION_TOOLBAR="Toolbar"
 TPL_ATUM_POSITION_TOP="Top"
+TPL_ATUM_POSITION_CUSTOMTOP="Custom Top"
```
Comment (Quy) [inline administrator/language/en-GB/tpl_atum.sys.ini]: Please sort language constant in alpha order.  Move it after line 8.

**Comment by richard67** [comment]
Why is it named “customtop” and not just e.g. “topmenu”?

**Comment by ceford** [comment]
`richard67: Why is it named “customtop”`
I thought it best not to imply that this is an alternative menu position for the standard menu. And it may have other uses. For example I often use a custom html module site notice 'this site is closing...'.

**Comment by infograf768** [comment]
Can be useful indeed.

**Comment by wilsonge** [comment]
I'm happy to merge this. I'm just adding here for future people seeing this. Just because this position exists doesn't mean we support a top menu. It means the position exists for people to use for whatever they want - if you choose to put a top menu there it's at your own risk ;)

**Comment by wilsonge** [comment]
Thanks!

**Comment by richard67** [comment]
Makes sense to me now, custom top, after the explanations above.

**Pull request merged**